prow/scripts/cluster-integration/kyma-gke-separate-compass-integration.sh

```diff
@@ -0,0 +1,379 @@
+}
+
+function installKyma() {
+  kymaUnsetVar=false
+
+  # shellcheck disable=SC2043
+  for var in GATEWAY_IP_ADDRESS ; do
+    if [ -z "${!var}" ] ; then
+      echo "ERROR: $var is not set"
+      kymaUnsetVar=true
+    fi
+  done
+  if [ "${kymaUnsetVar}" = true ] ; then
+    exit 1
+  fi
+
+  if [[ "$BUILD_TYPE" != "release" ]]; then
+    shout "Build Kyma-Installer Docker image"
+    date
+    CLEANUP_DOCKER_IMAGE="true"
+    KYMA_INSTALLER_IMAGE="${KYMA_INSTALLER_IMAGE}" "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}"/create-image.sh
+  fi
+
+  shout "Generate self-signed certificate"
+  date
+  DOMAIN="${DNS_SUBDOMAIN}.${DNS_DOMAIN%?}"
+  export DOMAIN
+  CERT_KEY=$("${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/generate-self-signed-cert.sh")
+  TLS_CERT=$(echo "${CERT_KEY}" | head -1)
+  TLS_KEY=$(echo "${CERT_KEY}" | tail -1)
+
+  shout "Apply Kyma config"
+  date
+
+  kubectl create namespace "kyma-installer"
+  applyCommonOverrides "kyma-installer"
+  applyKymaOverrides
+
+  if [[ "$BUILD_TYPE" == "release" ]]; then
+    echo "Use released artifacts"
+    gsutil cp "${KYMA_ARTIFACTS_BUCKET}/${RELEASE_VERSION}/kyma-installer-cluster.yaml" /tmp/kyma-gke-integration/downloaded-installer.yaml
+		"${KYMA_SCRIPTS_DIR}"/concat-yamls.sh /tmp/kyma-gke-integration/downloaded-installer.yaml "${INSTALLER_CR}" \
+		| kubectl apply -f-
+  else
+    echo "Manual concatenating yamls"
+    "${KYMA_SCRIPTS_DIR}"/concat-yamls.sh "${INSTALLER_YAML}" "${INSTALLER_CR}" \
+    | sed -e 's;image: eu.gcr.io/kyma-project/.*/installer:.*$;'"image: ${KYMA_INSTALLER_IMAGE};" \
+    | sed -e "s/__VERSION__/0.0.1/g" \
+    | sed -e "s/__.*__//g" \
+    | kubectl apply -f-
+  fi
+  
+  shout "Installation triggered"
+  date
+  "${KYMA_SCRIPTS_DIR}"/is-installed.sh --timeout 30m
+
+  if [ -n "$(kubectl get service -n kyma-system apiserver-proxy-ssl --ignore-not-found)" ]; then
+    shout "Create DNS Record for Apiserver proxy IP"
+    date
+    APISERVER_IP_ADDRESS=$(kubectl get service -n kyma-system apiserver-proxy-ssl -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+    APISERVER_DNS_FULL_NAME="apiserver.${DNS_SUBDOMAIN}.${DNS_DOMAIN}"
+    CLEANUP_APISERVER_DNS_RECORD="true"
+    IP_ADDRESS=${APISERVER_IP_ADDRESS} DNS_FULL_NAME=${APISERVER_DNS_FULL_NAME} "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/create-dns-record.sh"
+  fi
+}
+
+installCompass() {
+  kubectl create namespace "compass-installer"
+  applyCommonOverrides "compass-installer"
+  applyCompassOverrides
+
+  echo "Use master Compass artifacts"
+  readonly COMPASS_MASTER_ARTIFACTS="${COMPASS_DEVELOPMENT_ARTIFACTS_BUCKET}/master"
+  readonly COMPASS_TMP_DIR="/tmp/compass-gke-integration"
+  gsutil cp "${COMPASS_MASTER_ARTIFACTS}/compass-installer.yaml" ${COMPASS_TMP_DIR}/compass-installer.yaml
+  gsutil cp "${COMPASS_MASTER_ARTIFACTS}/is-installed.sh" ${COMPASS_TMP_DIR}/is-installed.sh
+	"${KYMA_SCRIPTS_DIR}"/concat-yamls.sh ${COMPASS_TMP_DIR}/compass-installer.yaml "${INSTALLER_CR}" \
```

Comment:
There's no need to concat component list with this artifact as it already contains one, also this `INSTALLER_CR` is for Kyma, not Compass installation.